src/libs/actions/Session/index.js

```diff
@@ -642,6 +621,23 @@ function authenticatePusher(socketID, channelName, callback) {
     });
 }
 
+function getShortLivedAuthToken() {
+    // TODO: If we have any way to read the shortLivedAuthToken and save it, we can also no longer call makeRequestWithSideEffects.
```

Comment:
I'm not sure what "and save it" implies. We are saving it in Onyx, but that isn't enough? Also, TODOs are not allowed in our code. Instead, you should open a GH issue and point to this line of code and then detail what needs to be done.